ansible_collections/arista/avd/molecule/cloudvision_tags/inventory/inventory.yml

```diff
@@ -0,0 +1,83 @@
+---
```

Comment:
I have attempted to reduce the inventory to the minimum topology that could be tested on cloudvision and still have 5-clos design for L3LS. Kindly let me know if it can be simplified further.